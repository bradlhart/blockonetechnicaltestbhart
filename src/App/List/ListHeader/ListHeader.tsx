import React, { useState } from 'react';

export const ListHeader: React.FC = () => {
  return (
    <div className="ui segment">
      <div className="ui three column divided grid" style={{ fontWeight: 'bold' }}>
        <div className="ten wide column">Hash</div>
        <div className="three wide column">Timestamp</div>
        <div className="three wide column">Actions</div>
      </div>
    </div>
  );
};
