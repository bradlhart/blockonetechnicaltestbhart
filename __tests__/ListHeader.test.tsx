import React from 'react';
import { shallow, ShallowWrapper } from "enzyme";

import ListHeader from '../src/App/List/ListHeader/ListHeader';

describe('ListHeader', () => {
    let wrapper: ShallowWrapper<any>;

    beforeEach(() => {
        wrapper = shallow(<ListHeader />);
    });

    test('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});