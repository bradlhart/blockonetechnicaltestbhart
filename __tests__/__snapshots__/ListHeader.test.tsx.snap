// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListHeader renders correctly 1`] = `
<div
  className="ui segment"
>
  <div
    className="ui three column divided grid listHeader listContent"
  >
    <div
      className="seven wide mobile eight wide tablet ten wide computer column"
    >
      Hash
    </div>
    <div
      className="five wide mobile five wide tablet four wide computer column"
    >
      Timestamp
    </div>
    <div
      className="four wide mobile three wide tablet two wide computer column"
    >
      Actions
    </div>
  </div>
</div>
`;
